contracts/scripts/anvil/state/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:22.04 as base_system
+
+# Update packages and install required dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y python3 python3-pip git nano curl iputils-ping redis-tools autossh nmap apt-utils supervisor
```

Comment:
Why installing so many dependencies?

Reply:
I deleted a few a seconds ago, but python, git and curl are necessary to install foundry or eigenlayer.